packages/math-input/src/components/input/math-input.tsx

```diff
@@ -328,11 +336,21 @@ class MathInput extends React.Component<Props, State> {
 
     blur: () => void = () => {
         this.mathField.blur();
-        this.props.onBlur && this.props.onBlur();
-        this.setState({focused: false, handle: {visible: false}});
+
```

Comment:
We don't want to be calling the props.onBlur method here as that automatically closes the keypad. This results in glitchy behaviour as clicking on a keypad button or tabbing away from the math input would result in the keypad closing.